tests/ecosystem/deployment/test_deployment.py

```diff
@@ -23,6 +23,11 @@ def test_deployment(pvc_factory, pod_factory):
                 'ocs_registry_image'
             )
             ocs_install_verification(ocs_registry_image=ocs_registry_image)
+
+            nb_eps = config.DEPLOYMENT.get('noobaa_endpoints')
+            if nb_eps > 1:
```

Comment:
maybe better to change:
'>' to '!=' in case we would like to test negative?

Reply:
this is in the deployment test. If we will want to test negative, it could be done in other tests